cpg-core/src/main/kotlin/de/fraunhofer/aisec/cpg/InferenceConfiguration.kt

```diff
@@ -57,14 +54,11 @@ private constructor(
 ) {
     class Builder(
         private var enabled: Boolean = true,
-        private var guessCastExpressions: Boolean = true,
         private var inferRecords: Boolean = true,
         private var inferFunctions: Boolean = true,
         private var inferVariables: Boolean = true,
         private var inferDfgForUnresolvedCalls: Boolean = true
     ) {
-        fun guessCastExpressions(guess: Boolean) = apply { this.guessCastExpressions = guess }
```

Comment:
It's not needed because we do not need to "guess" anymore in the frontend. The way it works now is that we "know" whether this a cast or not based on the information we have. We do not do any magic guessing anymore.